File: 2_BigO/src/py/main.py
from lc389 import *
from lc771 import *

def main():
    # Problem 389 test cases.
    print('Problem 389 test cases.')
    print(findTheDifference('abcd', 'abcde'))
    print(findTheDifference('', 'y'))
    print(findTheDifference('a', 'aa'))
    print(findTheDifference('ae', 'aea'))
    print()

    # Problem 771 test cases.
    print('Problem 771 test cases.')
    print(numJewelsInStones('aA', 'aAAbbbb'))
    print(numJewelsInStones('z', 'ZZ'))

if __name__ == '__main__':
    main()

File: 2_BigO/src/main.cpp
// 2020136018 김성녕

#include <iostream>

#include "lc389.h"
#include "lc771.h"

using namespace std;

int main(void) {
    // Problem 389 test cases.
    cout << "Problem 389 test cases.\n";
    cout << findTheDifference("abcd", "abcde") << endl;
    cout << findTheDifference("", "y") << endl;
    cout << findTheDifference("a", "aa") << endl;
    cout << findTheDifference("ae", "aea") << endl;
    cout << endl;

    // Problem 771 test cases.
    cout << "Problem 771 test cases.\n";
    cout << numJewelsInStones("aA", "aAAbbbb") << endl;
    cout << numJewelsInStones("z", "ZZ") << endl;

    return 0;
}
